src/server/scripts/Northrend/Naxxramas/boss_patchwerk.cpp

```diff
@@ -49,6 +49,14 @@ enum Misc
     ACHIEV_MAKE_QUICK_WERK_OF_HIM_STARTING_EVENT  = 10286
 };
 
+// These numbers are taken from anecdotal statements found on the interwebs.
+// Mainly based off of statements about tanks' TPS "going up by X amount on patchwerk" and then reverse calculating how much every hateful must give to reach this.
+// If you have actual numbers taken from retail, please update these!
+enum HatefulThreatAmounts {
+    HATEFUL_THREAT_10M = 2000,
+    HATEFUL_THREAT_25M = 2000,
```

Comment:
Intentional - separated in case they end up being different when/if someone updates with "real" data.
The compiler optimizes it out anyway if the values are the same.